tests/test.bats

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env bats
```

Comment:
Yay!

Reply:
It would be interesting to add arm64 testing (Github has the capabililty) but I wouldn't do it due to complexity if I were you.